ComposeStarter/app/build.gradle

```diff
@@ -80,8 +84,13 @@ dependencies {
 
     // Testing
     testImplementation libs.junit
+    testImplementation composeBom
```

Comment:
Maybe I don't need this actually since there is no compose test dependency for jvm tests 🤔